### PR TITLE
TODO List: Activity 1

### PR DESCRIPTION
**Pull request by TesheMaximillan**
# In this module♻️
** For a live preview please check the readme file **

- Set up a new project with webpack.
- Create an index.html file and write HTML markup here. 
- Create an empty To Do List placeholder (<ul> element). 
- The index.html file is set as a template using the HTML Webpack Plugin.
- Add fonts to webpack config
- ADD SCSS to webpack config

- index.js file is created and set an array of some simple to do tasks (array of objects) [microverse working time is used as a sample data] 

- Each task object contain three keys:
  - description [string].
  - completed [bool].
  - index: [number].
 
-  A function is written to iterate over the tasks array and populate an HTML list item element for each task.
- On page load render the dynamically created list of tasks in the dedicated placeholder. 
- The list appears in order of the index values for each task.
- Create a main.scss and set rules for the To-Do List. 
- SCSS is be loaded by Webpack Style/SCSS Loader.

### CODE REVIEW

**Review by keroloussamy — COMMENTED** [review]
Hi @TesheMaximillan, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have linters files. Kindly add them.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        use: [
+          'style-loader', // Inject styles into DOM
+          'css-loader', // Second-Step: Turns css into common js
+          'sass-loader', // First-Step: Turns sass into css
+        ],
+      },
+      // {
+      //   test: /\.css$/i,
+      //   use: ['style-loader', 'css-loader']
+      // },
```
Comment (elyor-doniyorov) [inline webpack.config.js]: - [ ] Kindly delete commented code because it makes your codebase unclean.

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @TesheMaximillan, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✔️ No linter errors
✔️ Webpack is used to set up the project
✔️ It has an array of some simple *to do tasks*

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — APPROVED** [review]
Hi, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**